src/Device/Driver/AR62xx.cpp

```diff
@@ -319,7 +319,7 @@ unsigned int AR62xxDevice::GenerateSetStationCommand(uint8_t *command, unsigned
   /** setting frequencies */
   command[command_length++] = 22;
 
-  uint16_t tmp[] = {0x00, 0x00};
+  uint16_t tmp[2] = {};
```

Comment:
Can you explain this change?

Reply:
looks better ;-)

Reply:
Thats your opinion, not mine. I disagree because using a default-initializer is rarely what you want - even if it happens to be the same in this special case.
But I question the existence of this whole initializer - why did you write one? And I question the existence of this variable. It's useless overhead.